### PR TITLE
Fix: Update line breaks

### PR DESCRIPTION
**Pull request by meysam81**

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/assign @sftim

File: content/en/docs/concepts/services-networking/endpoint-slices.md [hunk 1]
```diff
@@ -147,9 +147,9 @@ In the v1 API, the per endpoint `topology` was effectively removed in favor of
 the dedicated fields `nodeName` and `zone`.
 
 Setting arbitrary topology fields on the `endpoint` field of an `EndpointSlice`
-resource has been deprecated and is not be supported in the v1 API. Instead,
-the v1 API supports setting individual `nodeName` and `zone` fields. These
-fields are automatically translated between API versions. For example, the
+resource has been deprecated and is not going to be supported in the v1 API.
```
Comment (jihoon-seo) [inline content/en/docs/concepts/services-networking/endpoint-slices.md:150]: How about this?
(A tech review that checks whether this suggestion is correct or not is required.)

```suggestion
resource has been deprecated and is not supported in the v1 API.
```

Comment (sftim) [reply content/en/docs/concepts/services-networking/endpoint-slices.md:150]: That wording LGTM. I'm happy to skip tech review (but still recommend it) as the existing wording is at least as inaccurate.

Comment (reylejano) [reply content/en/docs/concepts/services-networking/endpoint-slices.md:150]: +1 to @jihoon-seo 's [suggestion](https://github.com/kubernetes/website/pull/31598#discussion_r797313656)

**Comment by tengqm** [comment]
@meysam81 Mind revise as per suggestions in the review comments?

**Pull request merged**